src/task/AttachMediaInputTask.ts

```diff
@@ -46,7 +46,7 @@ export default class AttachMediaInputTask extends BaseTask {
       }
     } else {
       await transceiverController.setAudioInput(null);
-      this.context.logger.warn('no audio track');
+      this.context.logger.info('no audio track');
```

Comment:
Should we check for audioHostURL?.length and only print out warn if it is !== 0 and just don't print in the other case.

Reply:
is the warning is useful ?